app/src/components/eventCard/style.module.scss

```diff
@@ -0,0 +1,49 @@
+$cont-width:250px;
+$cont-height: 40px;
+$cont-radius: 7px;
+$cont-padding: 7px;
+$cont-line-height: 1.1;
+$cont-bg:#F0EEEE;
+$font-size: 11px;
```

Comment:
plz use the variable in a separate file. see task F29
https://github.com/i-barysiuk/bstu-curators/pull/83/files